edysmp/stand-blog
# Add a custom profile and default content

## Issue
Fixes #…

## Description
Creates a custom profile called: `install_profile` and set up instead of `minimal`.

Added the `default_content` contrib module with two content: The contact form page and the the main menu link.

## Test Instructions
`ddev composer install`
`ddev si --existing-config`

After site install the Contact us menu link should be in the main menu along side the Home link.
One weborm node called `Contact us` should be set up with the default content.

## Approach (be as specific as possible)
With a custom profile we have more global control, e.g create general tests, or make custom logic for install modules.
Default content data used for demoing the progress and posibly it can work as the base content for the site.

## Screenshots



## Deployment
